### PR TITLE
Add changelogs page

### PR DESCRIPTION
**Pull request by Andre601**
This PR adds a Changelogs page for people to find.

It basically adds a new entry called "Changelogs" into the nav bar and has a page that links to the pages located within the changelogs folder.

A live-preview of this can be found at https://github.andre601.com/5zig-reborn.github.io/

**NOTE**: Due to how the domain stuff is handled on the website can you not just use the button, as this would turn the link into https://github.andre601.com/changelogs, which doesn't exists...
So you have to manually append "changelogs" to the URL. (https://github.andre601.com/5zig-reborn.github.io/changelogs/)

### CODE REVIEW

File: changelogs/index.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        <section class="jumbotron text-center">
+            <div class="container">
+                <h1 class="jumbotron-heading">The 5zig Mod - Changelogs</h1>
+                <p class="lead text-muted">All-in-one modification for Minecraft 1.8.9-1.15.2. <br>
+                    Revamp the way you play the game!</p>
+            </div>
+        </section>
+
+        <section class="text-center">
+            <h3>Changelogs</h3>
```
Comment (roccodev) [inline changelogs/index.html]: I'd change this to "Stable Changelogs". I'll eventually add a section for beta changelog with commit messages pulled from GitHub.

**Review by roccodev — CHANGES_REQUESTED** [review]
I left some comments in the files.  
Also, you're missing the navbar update in `faq/index.html`.

File: changelogs/index.html (hunk 1 above)
Comment (Andre601) [reply changelogs/index.html]: Changed to `Changelog (Stable)`

**Review by roccodev — CHANGES_REQUESTED** [review]
It's still missing the navbar update in faq/index.html.

**Comment by Andre601** [comment]
> It's still missing the navbar update in faq/index.html.

Didn't saw the comment. Updated it.

**Comment by Andre601** [comment]
Also, while we're at it I wanted to ask: Why have it all like `folder/index.html`?
Wouldn't it be better to have it just `faq.html`, `downloads.html`, etc?

From what I saw and know does GitHub Pages omit .html from pages (At least that's the case for my own projects page)

**Comment by roccodev** [comment]
Ah yes it does, that's nice.  
I was also thinking of using Jekyll templates so we don't have to manually copy/paste and update the same HTML multiple times.

**Review by roccodev — APPROVED** [review]

**Pull request merged**